docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,152 +334,110 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
-    --db-cluster-identifier $RDS_NAME \
-    --engine aurora-postgresql \
-    --engine-version $POSTGRESQL_VERSION \
-    --no-publicly-accessible \
-    --db-instance-class db.t3.medium
+kubectl patch storageclass gp2 -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
 ```
 
-More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-instance.html).
+After executing these commands, you will have a `gp3` StorageClass set as the default and the `gp2` StorageClass marked as non-default, provided that **gp2** was already present. Ensure to verify the changes again by running the `kubectl get storageclass` command.
 
-12. Wait for changes to be applied:
+### Requirements for a domain deployment
 
-```shell
-aws rds wait db-instance-available \
-    --db-instance-identifier $RDS_NAME
-```
+If you plan to deploy Camunda using an external domain associated with an external certificate, you will need to set up some IAM policies to allow both **external-dns** and **cert-manager** to interact with Route 53, which controls the DNS.
 
-### Verifying connectivity between the Amazon EKS cluster and the PostgreSQL database
+By default, the cluster uses **Pod Identity** to manage IAM roles for your applications. This means that service accounts are associated with IAM roles, allowing your pods to securely access AWS resources without hardcoding credentials. For more information on configuring Pod Identity, refer to the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html).
 
-1. Retrieve the writer endpoint of the DB cluster.
+Additionally, to [enable OpenID Connect (OIDC) and IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) on your cluster, you can use the following commands:
 
-```shell
-export DB_HOST=$(aws rds describe-db-cluster-endpoints \
-  --db-cluster-identifier $RDS_NAME \
-  --query "DBClusterEndpoints[?EndpointType=='WRITER'].Endpoint" \
-  --output text)
-```
+1. **Determine the OIDC issuer ID for your cluster:**
 
-2. Start Ubuntu container in interactive mode within the Amazon EKS cluster.
+   First, ensure that your EKS cluster is set up with an OIDC provider. The following command should show you the OIDC issuer:
 
-```shell
-kubectl run ubuntu --rm -i --tty --image ubuntu --env="DB_HOST=$DB_HOST" --env="PG_USERNAME=$PG_USERNAME" -- bash
-```
+   ```bash
+   export oidc_id=$(aws eks describe-cluster --name "$CLUSTER_NAME" --query "cluster.identity.oidc.issuer" --output text)
+   echo "$oidc_id"
+   ```
 
-3. Install required dependencies:
+   Determine whether an IAM OIDC provider with your cluster’s issuer ID is already in your account.
 
-```shell
-apt update && apt install -y postgresql-client
-```
+   ```bash
+   aws iam list-open-id-connect-providers | grep $oidc_id | cut -d "/" -f4
+   ```
 
```

Comment:
I was having a problem with this one detecting anything.

`oidc_id=https://oidc.eks.eu-central-1.amazonaws.com/id/3C1A33F675599F7B5AD3C663E2CC2DCE`

```
{
    "OpenIDConnectProviderList": [
        {
            "Arn": "arn:aws:iam::444804106854:oidc-provider/oidc.eks.eu-central-1.amazonaws.com/id/3C1A33F675599F7B5AD3C663E2CC2DCE"
        },
        {
            "Arn": "arn:aws:iam::444804106854:oidc-provider/oidc.eks.eu-west-2.amazonaws.com/id/0184EE512F7947A1EDABDCFCC69F158F"
        },
        {
            "Arn": "arn:aws:iam::444804106854:oidc-provider/oidc.eks.eu-west-2.amazonaws.com/id/4EBF8FD07659D7F8F4BC2DC11F9A282C"
        },
        {
            "Arn": "arn:aws:iam::444804106854:oidc-provider/oidc.eks.eu-west-2.amazonaws.com/id/6B0023DBBADEB00479FF98F82F8CC427"
        },
        {
            "Arn": "arn:aws:iam::444804106854:oidc-provider/oidc.eks.eu-west-2.amazonaws.com/id/B6946796624DBD2C57423647D969FF7F"
        }
    ]
}
```

---

as discussed just `| cut -d '/' -f 5`, see official docs again.